Cargo.toml

```diff
@@ -62,6 +62,7 @@ ordered-float = { version = "3.7" }
 getrandom = { version = "0.2.10", features = ["js"], optional = true }
 
 im-rc = "15.1.0"
```

Comment:
`im-rc` is not used anywhere in the codebase?

Reply:
It is, it's just imported as `im_rc` so might be missed on a quick find... It's used in the constraint solver.